test/main.cpp

```diff
@@ -27,6 +39,216 @@ const char ec_public_2[] = "-----BEGIN PUBLIC KEY-----\n"
 "XWCq4E/g2ME/uBOdP8RE0tqle8fxYcaPikgMcppGq2ycTiLGgEYXgsq2JA==\n"
 "-----END PUBLIC KEY-----\n";
 
+/**
+ * Duplicate of get_cache_file from scitokens_cache.cpp; used for direct
+ * SQLite manipulation.
```

Comment:
What’s the purpose of these duplicate functions rather than calling the library functions?